system-addon/test/unit/lib/Screenshots.test.js

```diff
@@ -102,4 +131,23 @@ describe("Screenshots", () => {
       assert.propertyVal(str, 999999, "\u0009");
     });
   });
+
+  describe("#_shouldGetScreenshots", () => {
+    beforeEach(() => {
+      let more = 2;
+      sandbox.stub(global.Services.wm, "getEnumerator").returns({getNext: () => {}, hasMoreElements() { return more--; }});
+    });
+    it("should use private browsing utils to determine if a window is private", () => {
+      Screenshots._shouldGetScreenshots();
+      assert.calledOnce(global.PrivateBrowsingUtils.isWindowPrivate);
+    });
+    it("should return true if there exists at least 1 non-private window", () => {
+      global.PrivateBrowsingUtils.isWindowPrivate = () => false;
+      assert.isTrue(Screenshots._shouldGetScreenshots());
+    });
+    it("should return false if there exists private windows", () => {
+      global.PrivateBrowsingUtils.isWindowPrivate = () => true;
```

Comment:
Let's check that this gets called the expected number of times. Then good to land!